tests/test-data/waf-collection-source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "waf-collection-source-$RNDCODE",
+    "url": "http://test.com-$RNDCODE",
+    "source_type": "waf-collection",
+    "title": "WAF test $RNDCODE",
+    "collection_metadata_url": "http://coll.test.com-$RNDCODE",
+    "frequency": "MANUAL",
+    "owner_org": "test-organization-$RNDCODE"
+}
```

Comment:
With such a few lines, I think it's easier to read if this was just inline within the test.